server/graphql/v1/mutations/orders.js

```diff
@@ -119,6 +120,13 @@ async function checkRecaptcha(order, remoteUser, reqIp) {
   return response;
 }
 
+function fundExceededLimit(addedFunds, addedFundsLimit) {
```

Comment:
Sorry, can you put it back as it was? My feedback was not clear

Reply:
Okay.